tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -334,17 +335,17 @@ these properties, and if you don't write to them, Godot will optimize away the c
 |                                        | responsible for setting the ``DEPTH`` for **all** other branches. Otherwise, the graphics API    |
 |                                        | will leave them uninitialized.                                                                   |
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
-| inout vec3 **NORMAL**                  | Normal that comes from vertex function (default, in view space).                                 |
+| inout vec3 **NORMAL**                  | Normal that comes from the ``vertex()`` function (default, in view space).                       |
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
-| inout vec3 **TANGENT**                 | Tangent that comes from vertex function.                                                         |
+| inout vec3 **TANGENT**                 | Tangent that comes from the ``vertex()`` function (default, in view space).                      |
 +----------------------------------------+--------------------------------------------------------------------------------------------------+
-| inout vec3 **BINORMAL**                | Binormal that comes from vertex function.                                                        |
+| inout vec3 **BINORMAL**                | Binormal that comes from the ``vertex()`` function (default, in view space).                     |
```

Comment:
Unconfirmed, based on the description of `skip_vertex_transform` and `world_vertex_coords`.

Reply:
👍